projects/igniteui-angular/src/lib/combo/combo-item.component.ts

```diff
@@ -50,7 +50,7 @@ export class IgxComboItemComponent extends IgxDropDownItemBase {
      * @hidden
      */
     get isSelected() {
-        return this.parentElement.selectedItem.indexOf(this.itemID) > -1;
+        return this.parentElement.parentElement.isItemSelected(this.itemID);
```

Comment:
My comment is still not addressed.

Reply:
Isn't it addressed into https://github.com/IgniteUI/igniteui-angular/pull/2028/ ?
I think the two comments came one after another and there was misunderstanding

Reply:
@bazal4o Why would a PR review be addressed in a different PR?

Reply:
Along with the combo item I also refactored the combo drop down, so I've decided to create different branch, because I wanted to separate the the logic for selection and the renaming of the injectable components. I can now merge both here, if you want.

Reply:
@Lipata Please do.